test/src/test/java/jenkins/model/NodesRestartTest.java

```diff
@@ -0,0 +1,47 @@
+package jenkins.model;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertNotNull;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.model.Descriptor;
+import hudson.model.Slave;
+import hudson.slaves.ComputerLauncher;
+import java.io.IOException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+public class NodesRestartTest {
+    @Rule
+    public RestartableJenkinsRule s = new RestartableJenkinsRule();
```

Comment:
We generally prefer `JenkinsSessionRule` to `RestartableJenkinsRule` unless there is a particular reason to use `RestartableJenkinsRule`. I see no particular reason to use it in this case.